### PR TITLE
Add downloadable comprehensive preview

### PR DESCRIPTION
**Pull request by cansavvy**
## Summary 

Was trying to think of a way that a more comprehensive preview could be shown. Because at times I want to check for a change that https://htmlpreview.github.io can't handle -- dropdowns and font-awesome icons for example don't get shown. 

In this set up a rendered version of the website gets rendered as usually but a copy of it gets zipped up and archived so people could download it.

### CODE REVIEW

**Comment by cansavvy** [comment]
It works!

**Comment by carriewright11** [comment]
Nice! this is helpful for checking the rendered version. Where is it getting archived? is it temporary for while the PR is open?

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -136,9 +141,11 @@ jobs:
         run: |
           course_name=$(head -n 1 _website.yml | cut -d'"' -f 2| tr " " "-")
           website_link=$(echo "https://htmlpreview.github.io/?https://raw.githubusercontent.com/$GITHUB_REPOSITORY/preview-${{ github.event.pull_request.number }}/docs/index.html")
-          echo ::set-output name=website_link::$website_link
-          echo ::set-output name=time::$(date +'%Y-%m-%d')
-          echo ::set-output name=commit_id::$GITHUB_SHA
+          docs_link=$(echo "https://github.com/$GITHUB_REPOSITORY/raw/preview-${{ github.event.pull_request.number }}/website-preview.zip")
```
Comment (avahoffman) [inline .github/workflows/pull_request.yml:144]: can we name this something more descriptive, like "zip_file" ? docs_link makes it sound like its gonna take you to documentation

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -136,9 +141,11 @@ jobs:
         run: |
           course_name=$(head -n 1 _website.yml | cut -d'"' -f 2| tr " " "-")
```
Comment (avahoffman) [inline .github/workflows/pull_request.yml:142]:
```suggestion
```
appears not to be used. Did you mean to echo it to the log?

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (avahoffman) [reply .github/workflows/pull_request.yml:144]: Could we also break this chunk up into "# Store variables" and "# Print to log" for readability?

**Review by avahoffman — APPROVED** [review]
Minor comments! lgtm!

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (cansavvy) [reply .github/workflows/pull_request.yml:144]: Will do! On both accounts!

Comment (cansavvy) [reply .github/workflows/pull_request.yml:144]: Oh actually the "echo to log" can't really be separated the `echo "zip_link=$docs_link" >> $GITHUB_OUTPUT` aren't echoing to log, the are saving the GitHub variables to the output and have to be done in the same step.

File: .github/workflows/pull_request.yml (hunk 2 above)
Comment (cansavvy) [reply .github/workflows/pull_request.yml:142]: Deleted!

**Comment by cansavvy** [comment]
> Nice! this is helpful for checking the rendered version. Where is it getting archived? is it temporary for while the PR is open?

It is temporary while the PR is open!

**Pull request merged**